src/applications/appeals/995/components/PrimaryPhone.jsx

```diff
@@ -62,7 +62,11 @@ export const PrimaryPhone = ({
   };
 
   const navButtons = onReviewPage ? (
-    <va-button text={content.update} onClick={updatePage} uswds />
+    <va-button
+      text={content.update}
+      onClick={updatePage}
+      label="update primary phone number"
+    />
```

Comment:
This change doesn't seem related to the rest of this PR?

Reply:
It fixes an accessibility issue with the "Update page" button on the review and submit page. Without this change, screen readers would only read out the text "Update page" - so it's sorta related in that I found it while testing focus management

Reply:
Yeah, all web components are now using USWDS v3, so the prop setting isn't required any more

Reply:
Nice, thanks!